### ISSUE TITLE
Cant use PoB

### ISSUE THREAD

**Issue by Daamon0700** [Openarl/PathOfBuilding#847]
Trying to open Path of Building and getting an error message saying "The code execution cannot proceed because MSVCR100.dll was not found. Reinstalling the program may fix this problem." Already tried deleting entire folder and redownloading and unzipping, did not help.

**Comment by Thirsk** [comment]
Do you have Microsoft Visual C++ 2010 installed on your computer? Because that's what the error is indicating you're missing.

**Comment by Daamon0700** [comment]
Yeah it was installed, reinstalled it though and it fixed the issue, so thats wierd, but thanks :)
